src/api.v2/helpers/worker/workSubmit/createWorkerK8s.js

```diff
@@ -62,7 +102,21 @@ const createWorkerResources = async (service) => {
     return { name, creationTimestamp, phase };
   }
 
-  const pods = await getAvailablePods(namespace);
+  // scale pods if worker has zero replicas
+  let pods = await getAvailablePods(namespace);
+  let replicas;
+  const minDesiredReplicas = 1;
+  try {
+    ({ spec: { replicas } } = await getDeployment('worker', namespace));
+  } catch (e) {
+    logger.log('Getting worker deployment failed, ignoring...', e);
+  }
+
+  if (pods.length < 1 && replicas < minDesiredReplicas) {
```

Comment:
Note: `replicas < minDesiredReplicas` evaluates to `false` if the above trycatch fails (in which case replicas is still undefined)